projects/lib/src/admin/components/page-list/page-list.component.ts

```diff
@@ -98,6 +106,33 @@ export class PageListComponent implements OnInit, OnChanges {
            changes[propertyName].previousValue !== changes[propertyName].currentValue;
   }
 
+  changeSortStrategy(sortBy: string) {
+    if (this.sortBy && this.sortBy === sortBy){
+      sortBy = "!" + this.sortBy;
+      this.sortBy = sortBy;
+    }
+    else {this.sortBy = sortBy}
```

Comment:
you don't need this else statement here. if you move `this.sortBy = sortBy;` out of the if and remove the else it will work the same way